libcodechecker/source_code_comment_handler.py

```diff
@@ -193,8 +193,9 @@ def get_source_line_comments(self, bug_line):
             if has_any_marker:
                 rev = list(reversed(curr_suppress_comment))
                 suppress_comment = ''.join(rev).replace('//', '')
-                source_line_comments.append(
-                    self.__process_source_line_comment(suppress_comment))
+                comment = self.__process_source_line_comment(suppress_comment)
+                if comment:
```

Comment:
Maybe we can print a warning message to the user, that a wrongly formatted source code comment can be found in the source code.